src/components/Game/Game.constants.js

```diff
@@ -149,6 +150,126 @@ const ironMaterial = {
   price: 5,
 }
 
-export const materials = [goldMaterial, silverMaterial, ironMaterial]
+const zincMaterial = {
+  name: 'Zinc',
+  image: zinc,
+  price: 5,
+}
+
+const copperMaterial = {
+  name: 'Copper',
+  image: copper,
+  price: 5,
+}
+
+// //////
+
+const bronzeMaterial = {
+  name: 'Bronze',
+  image: bronze,
+  price: 100,
+}
+
+const whiteGoldMaterial = {
+  name: 'White Gold',
+  image: whiteGold,
+  price: 100,
+}
+
+const chineseSilverMaterial = {
+  name: 'Chinese Silver',
+  image: chineseSilver,
+  price: 100,
+}
+
+const billonMaterial = {
+  name: 'Billon',
+  image: billon,
+  price: 20,
+}
+
+const whiteGoldRecipe = {
+  name: 'White Gold Recipe',
+  necessaryMaterials: [
+    {
+      material: goldMaterial,
+      count: 2,
```

Comment:
I think count is not a `happy` name for that property, quantity, quantityNeeded or countNeeded, would be a better name